openapi_python_client/utils.py

```diff
@@ -36,27 +37,21 @@ def remove_string_escapes(value: str) -> str:
     return value.replace('"', r"\"")
 
 
-def to_valid_python_identifier(value: str) -> str:
-    """
-    Given a string, attempt to coerce it into a valid Python identifier.
-
-    If valid, return it unmodified.
+# This can be changed by config.Config.load_config
+FIELD_PREFIX = "field"
 
-    If invalid, prepend a fixed prefix. This resolves some problems caused by the string's leading
-    character.
 
-    If that prefix does not make it a valid identifier - there are unsupported non-leading
-    characters - raise a ValueError.
+def to_valid_python_identifier(value: str) -> str:
+    """
+    Given a string, attempt to coerce it into a valid Python identifier by stripping out invalid characters and, if
+    necessary, prepending a prefix.
 
     See:
         https://docs.python.org/3/reference/lexical_analysis.html#identifiers
     """
-    if value.isidentifier():
-        return value
-
-    new_value = f"field_{value}"
+    new_value = fix_keywords(sanitize(value))
 
     if new_value.isidentifier():
         return new_value
 
-    raise ValueError(f"Cannot convert {value} to a valid python identifier")
+    return f"{FIELD_PREFIX}_{new_value}"
```

Comment:
1. I'd suggest including the `_` in the prefix itself (i.e. `field_`) rather than here, so the user can opt to omit it

2. This could still be an invalid identifier - one case being that `FIELD_PREFIX` itself has invalid characters. This could be checked here, in `Config`, or not at all

Reply:
Good call, I've made the config change so people can omit the underscore if they want. For now I don't want to try and catch user-supplied errors, I'm pretty sure invalid classes can also be provided from the other config values in the current version.